frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/extension.dart

```diff
@@ -112,7 +146,17 @@ class SelectOptionTag extends StatelessWidget {
               name,
               fontSize: FontSizes.s11,
               overflow: TextOverflow.ellipsis,
-              color: AFThemeExtension.of(context).textColor,
+              color: (calculateContrast(
+                        color,
+                        Theme.of(context)
+                            .extension<AFThemeExtension>()!
+                            .textColor,
+                      ) <
+                      1.36)
+                  ? Theme.of(context)
+                      .extension<AFThemeExtension>()!
+                      .textColorAlt
+                  : Theme.of(context).extension<AFThemeExtension>()!.textColor,
```

Comment:
This should be done by a method, consider adding a helper method to the util class (see above comment).

That takes backgroundColor, primaryTextColor, and altTextColor as the arguments, and return the proper text color.